src/shared/common/components/workstack-tag.jsx

```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+export default function Tags(props) {
+    let tags = [];
+
+    if (props.row.tag && props.row.tag.length > 0) {
+        tags = props.row.tag.map(tag => {
+            return <span key={tag} className="govuk-tag govuk-!-margin-right-1">{tag}</span>;
```

Comment:
consider `<abbr>`

Reply:
Currently we only have one tag which is an abbreviation I guess but not sure if we guarantee/enforce all future tags should be abbreviations?